cluster-provision/images/kubevirt-testing/kernel-boot-container/Dockerfile.alpine-ext-kernel-boot-demo

```diff
@@ -0,0 +1,5 @@
+FROM alpine:latest
+RUN apk update
+RUN apk add linux-virt
+RUN apk add openrc
+
```

Comment:
We need to change the `initramfs-virt` and `vmlinuz-virt` accessible for `qemu`.  Can we change the ownership?

Reply:
Done, PR is updated.
Also, made a PR that will use this new container version:
https://github.com/kubevirt/kubevirt/pull/5779